x-pack/plugins/secops/public/components/flyout/header/index.tsx

```diff
@@ -29,11 +28,12 @@ interface OwnProps {
 
 interface StateReduxProps {
   description?: string;
+  getNotesByIds: (eventIds: string[]) => Note[];
```

Comment:
consider renaming this to `getNotesByEventIds`, because this behavior is not revealed unless you are looking at  the name of the `eventIds` parameter

Reply:
I am really getting notes by their ID since `eventIdToNotesId` is giving me the ids

![image](https://user-images.githubusercontent.com/189600/52432755-4b3bf080-2ad9-11e9-8430-f03aee49bcfb.png)